### PR TITLE
Update how-to-update-handbook.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Changed heading capitalization. Removed 1x offboarded staff member.

### CODE REVIEW

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 1]
```diff
@@ -95,18 +95,17 @@ Follow these two steps:
...
 [Watch a training video on how to update the handbook in GitHub](https://drive.google.com/file/d/1AOI8H-oe2u1JW6oOA4nPPTSbGnK3Xuq1/view?usp=sharing).
 
-## Approved Reviewers and Permissions
+## Approved reviewers and permissions
 
 Below is a list of approved reviewers.
 
 1. @jasonblais: Reviews major changes to handbook.mattermost.com, such as updates to the Table of Contents (SUMMARY.md).
 2. @justinegeffen, @amyblais, @cwarnermm: Editor reviews of all submitted PRs for correct grammar and consistent style.
-3. @rbradleyhaas: Signs off on changes to [business operations](https://handbook.mattermost.com/operations/business-operations).
```
Comment (jasonblais) [inline company/how-to-guides-for-staff/how-to-update-handbook.md:108]: Can you touch base with Kevin Fayle if he'd like to be the sign-off person? He's the new Dir of Marketing Ops and Analytics, and might like to have an opportunity to review.

He only started yesterday, but I don't foresee changes in the coming weeks that he'd have to review

**Review by jasonblais — CHANGES_REQUESTED** [review]
Just one comment above, otherwise no concerns

File: company/how-to-guides-for-staff/how-to-update-handbook.md (hunk 1 above)
Comment (justinegeffen) [reply company/how-to-guides-for-staff/how-to-update-handbook.md:108]: Good idea, and he's happy to do so. I've made that change. :)

Comment (jasonblais) [reply company/how-to-guides-for-staff/how-to-update-handbook.md:108]: Sorry, I forgot the old BizOps section got removed via https://github.com/mattermost/mattermost-handbook/pull/687/files

For now, let's link to https://handbook.mattermost.com/operations/messaging-and-math/demand-generation-reporting instead

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 2]
```diff
@@ -95,17 +95,17 @@ Follow these two steps:
...
+3. @kevinfayle: Signs off on changes to [marketing ops and analytics](https://handbook.mattermost.com/operations/research-and-development/product/analytics).
```
Comment (jasonblais) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]:
```suggestion
3. @kevinfayle: Signs off on changes to [marketing ops and analytics](https://handbook.mattermost.com/operations/messaging-and-math/demand-generation-reporting).
```

**Review by jasonblais — CHANGES_REQUESTED** [review]

**Review by jasonblais — APPROVED** [review]

**Pull request merged**